### PR TITLE
[DRAFT] ki landing page

### PR DESCRIPTION
**Pull request by goatsOnDrugs**
## Description

WIP
- created secondary buttons and added "blue" variant to both
- created a page for ki landing page and did the first section

UP NEXT:
1. switch "Enter app" button with "contact sales"
2. implement dark mode by default
3. add the blue fade to the background
4. next section: create cards
5. make cards scroll nice
## Related Ticket

Resolves #<issue-number>

## Checklist

- [ ] Building the site with `npm run build-site` works without errors
- [ ] Building the app with `npm run build-app` works without errors
- [ ] I formatted JS and TS files with running `npm run format-all`

### CODE REVIEW

**Comment by goatsOnDrugs** [comment]
image

File: site/components/pages/Infinity/index.tsx [hunk 1]
```diff
@@ -0,0 +1,705 @@
...
+              <Text t="body7">
+                The next-generation carbon toolkit for your organization
+              </Text>
+            </Trans>
+          </div>
+          <div className="cta_right_container">
+            <ButtonPrimary
+              label="Get Started"
+              variant="blueRounded"
+              href="https://klima-dao.notion.site/Welcome-to-Klima-Infinity-f2626cb26a474e879287038662e83e47"
```
Comment (ladytrekker) [inline site/components/pages/Infinity/index.tsx:692]: I would put all links related to anything with KlimaDAO to the constants file

File: site/components/pages/Infinity/index.tsx [hunk 2]
```diff
@@ -0,0 +1,705 @@
...
+            />
+            <ButtonSecondary
+              label="Contact sales"
+              variant="blueRounded"
+              href="https://notionforms.io/forms/klimadao-request-for-collaboration"
```
Comment (ladytrekker) [inline site/components/pages/Infinity/index.tsx:697]: This one too

File: site/components/pages/Infinity/index.tsx [hunk 3]
```diff
@@ -0,0 +1,705 @@
...
+                </Text>
+              </Trans>
+            </div>
+          </div>
+          <div className="faq_button_container">
+            <ButtonPrimary
+              className="faq_button"
+              variant="blueRounded"
+              label="Read in-depth FAQs for KI Clients"
+              href="https://www.klimadao.finance/blog/klima-infinity-faqs"
```
Comment (ladytrekker) [inline site/components/pages/Infinity/index.tsx:665]: This one too to constants

File: site/components/pages/Infinity/index.tsx [hunk 4]
```diff
@@ -0,0 +1,705 @@
+import React, { useRef, useState } from "react";
+import { NextPage } from "next";
+import Image from "next/image";
+// import Link from "next/link";
```
Comment (ladytrekker) [inline site/components/pages/Infinity/index.tsx:4]: This line can be removed

File: site/components/pages/Infinity/index.tsx [hunk 5]
```diff
@@ -0,0 +1,705 @@
...
+                Klima Infinity is a next-generation carbon toolkit for your
+                organization
+              </Text>
+            </Trans>
+            <div className="hero_buttons">
+              <ButtonPrimary
+                className="hero_button_primary"
+                variant="blueRounded"
+                label="Get started"
+                href="https://klima-dao.notion.site/Welcome-to-Klima-Infinity-f2626cb26a474e879287038662e83e47"
```
Comment (ladytrekker) [inline site/components/pages/Infinity/index.tsx:128]: Here too

File: site/components/pages/Infinity/index.tsx [hunk 6]
```diff
@@ -0,0 +1,705 @@
...
+              />
+              <ButtonSecondary
+                variant="blueRounded"
+                label="Contact sales"
+                href="https://notionforms.io/forms/klimadao-request-for-collaboration"
```
Comment (ladytrekker) [inline site/components/pages/Infinity/index.tsx:133]: Same

File: site/components/pages/Infinity/cards.ts [hunk 7]
```diff
@@ -0,0 +1,69 @@
+import olympusLogo from "public/logo-olympus-light.png";
+import polygonLogo from "public/logo-polygon-white.png";
+import markCubanLogo from "public/logo-mark-cuban.png";
+import stadiusLogo from "public/logo-stadius.png";
+
```
Comment (ladytrekker) [inline site/components/pages/Infinity/cards.ts:5]: Just for clarification: All external links int his file are not KlimaDao related and do not have to moved to constants IMO

**Review by ladytrekker — APPROVED** [review]
Thanks. I can feel the sweat which comes when dealing with Next and its Images 👍 

I can see a lot of refactorings. But as @Atmosfearful pointed out this can be done in follow-up PRs, everything looks so far fine in my eyes.

I did not do intensive browser testings though and would recommend to wait for someone to do that first before merging to staging.

**Review by Atmosfearful — CHANGES_REQUESTED** [review]
Page looks awesome! 🎉 
Code is looking pretty solid too, here are few bigger issues to address now tho-- esp. the text/trans tag order because that affects what translators will see when we merge to staging.

Also:
* [ ] For images that are not transparent, and don't take up a lot of screen space, try converting from png to jpg with `convertio.co` or another tool. That seems to shave ~30% off the file size. But consider that *sometimes* for very large images with gradients or other details, the jpg looks noticeably worse. It's tedious but saves us a ton of bandwidth!
* [ ] related to the above, I see that some pngs have transparent rounded corners. If you can, import the rectangular image as a `jpg`, and apply border radius with css instead

File: site/components/pages/Infinity/index.tsx [hunk 8]
```diff
@@ -0,0 +1,736 @@
...
+              className="carousel_image"
+            />
+            <Trans id="infinity.carousel_image_description">
+              <Text t="body3" className="carousel_image_description">
+                Wind power project at Jaibhim, India
+              </Text>
+            </Trans>
+          </div>
+          <div className="carousel_right_container">
+            <Trans id="infinity.carousel_info">
```
[the remark this answers is not in the document]
Comment (goatsOnDrugs) [reply site/components/pages/Infinity/index.tsx]: updated! I will never make that mistake again lol

File: site/components/pages/Infinity/index.tsx [hunk 9]
```diff
@@ -0,0 +1,736 @@
...
+            >
+              <Trans id="faq.answer2">
+                <Text t="body6">
+                  Carbon credits are a high-integrity mechanism that allow
+                  carbon finance to flow from those who pollute, to projects
+                  across the globe that help to mitigate or remove carbon from
+                  the atmosphere. They are a valuable tool to cover
+                  hard-to-abate emissions that may be costly or technically
+                  challenging to eliminate today.
+                  <br />
```
[the remark this answers is not in the document]
Comment (goatsOnDrugs) [reply site/components/pages/Infinity/index.tsx]: good point. updated,

**Comment by ladytrekker** [comment]
Closing in favor of https://github.com/KlimaDAO/klimadao/pull/550

**Pull request closed without merging**